### PR TITLE
:white_check_mark: Overview section fixes#4

### PR DESCRIPTION
**Pull request by Bsquiat1**
# Description

I added a section that includes the image and description provided together with styling

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshot (optional)

desktop view
Screenshot from 2023-10-03 15-36-56

mobile view

Screenshot from 2023-10-05 09-54-04

tablet view

Screenshot from 2023-10-05 09-54-55

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code

- [x] My changes generate no new warnings

- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: index.html
Comment (paulkahura) [inline index.html:1]: Add tablet screenshot. Otherwise great job

**Review by paulkahura — CHANGES_REQUESTED** [review]

File: index.html
Comment (Bsquiat1) [reply index.html:1]: Thank you. I have added tablet screenshot

**Review by paulkahura — CHANGES_REQUESTED** [review]
Also use rem/em instead of px. There are online calculators. Also the margin between the text and image is abit off. Otherwise looks good to me.

File: index.html
Comment (Bsquiat1) [reply index.html:1]: I have made the changes

**Comment by JAPHETHNYARANGA** [comment]
Good attempt but this issue has been closed . Please pick another open issue. Hint; don't delete code already in a file when working on it.

**Pull request closed without merging**